packages/api/endpoints/granules.js

```diff
@@ -213,20 +215,28 @@ const create = async (req, res) => {
  * Update existing granule *or* create new granule
  *
  * @param {Object} req - express request object
+ * @param {number} req.dbMaxPool - optional max pool size for knex
+ * @param {Object} req.testContext - test context for client requests
+ * @param {Object} req.body - request body for patching a granule
  * @param {Object} res - express response object
  * @returns {Promise<Object>} promise of an express response object.
  */
-const patchGranule = async (req, res) => {
+const patchGranuleMethod = async (req, res) => {
```

Comment:
This looks like the abstraction to remove `_returnPatchGranuleStatus` from `patchGranule`, but the difference in name between `patchGranuleMethod` and `patchGranule` is probably confusing without more context.    The docstrings also have the exact same header text. 

Consider something like `patchGranule` for this method and `patchGranuleAndReturnStatus` or something along those lines instead.

Reply:
😍  looks good, thank you!